pkg/util/permissions/permissions_test.go

```diff
@@ -0,0 +1,74 @@
+package permissions
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/authorization/mgmt/2015-07-01/authorization"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+func TestCanDoAction(t *testing.T) {
+	tests := []struct {
+		name        string
+		permissions []authorization.Permission
+		action      string
+		want        bool
+		wantErr     bool
+	}{
+		{
+			name:        "empty permissions list",
+			permissions: []authorization.Permission{},
+			action:      "Microsoft.Network/virtualNetworks/subnets/join/action",
+			want:        false,
```

Comment:
also please omit zero values like false

Reply:
I will remove, but I don't think it's a good idea. I include zero values/empty/etc slices in tests because cases like this look very weird:

```go
{
    name:   "empty permissions list",
    action: "Microsoft.Network/virtualNetworks/subnets/join/action",
},
```

What is expected value? Where is the list of permissions? More verbose version os more self-explanatory. IMHO, DRY principle - is an enemy in testing.

Reply:
Just to clarify: I have opinions, but I'm happy to do whatever the team decides is best. But would be great to have tooling to check code style. Should save us some time too.

Reply:
I think I have come to like this style of omitting zero values too.

Reply:
yeah, this is a particularly bad example - usually when you say `wantErr` the meaning is clearer.